### PR TITLE
Fix X500Name, RPC api usage

### PR DESCRIPTION
**Pull request by exFalso**

### CODE REVIEW

File: kotlin-source/src/main/kotlin/com/template/Main.kt [hunk 1]
```diff
@@ -24,11 +25,11 @@ fun main(args: Array<String>) {
...
     val user = User("user1", "test", permissions = setOf())
     driver(isDebug = true) {
-        startNode("Controller", setOf(ServiceInfo(ValidatingNotaryService.type)))
+        startNode(getX509Name("Controller", "London", "root@city.uk"), setOf(ServiceInfo(ValidatingNotaryService.type)))
         val (nodeA, nodeB, nodeC) = Futures.allAsList(
-                startNode("NodeA", rpcUsers = listOf(user)),
-                startNode("NodeB", rpcUsers = listOf(user)),
-                startNode("NodeC", rpcUsers = listOf(user))).getOrThrow()
+                startNode(getX509Name("NodeA", "Paris", "root@city.fr"), rpcUsers = listOf(user)),
+                startNode(getX509Name("NodeB", "Rome", "root.city.it"), rpcUsers = listOf(user)),
```
Comment (rnicoll) [inline kotlin-source/src/main/kotlin/com/template/Main.kt]: `root@city.it` perhaps? Also I know this isn't going to look as good, but I'd rather use reserved domains (as per https://tools.ietf.org/html/rfc2606 ) that can never be valid. So perhaps:

root@city.fr.example
root@city.it.example
root@city.us.example

**Comment by exFalso** [comment]
Back to you

File: java-source/src/main/java/com/template/Main.java [hunk 2]
```diff
@@ -32,16 +32,16 @@ public static void main(String[] args) {
...
+                            singleton(new ServiceInfo(ValidatingNotaryService.Companion.getType(), null)),
                             emptyList(),
                             VerifierType.InMemory,
                             emptyMap());
 
                     try {
-                        NodeHandle nodeA = dsl.startNode("NodeA", emptySet(), ImmutableList.of(user), VerifierType.InMemory, emptyMap()).get();
-                        NodeHandle nodeB = dsl.startNode("NodeB", emptySet(), ImmutableList.of(user), VerifierType.InMemory, emptyMap()).get();
-                        NodeHandle nodeC = dsl.startNode("NodeC", emptySet(), ImmutableList.of(user), VerifierType.InMemory, emptyMap()).get();
+                        NodeHandle nodeA = dsl.startNode(getX509Name("NodeA", "Paris", "root@city.fr"), emptySet(), ImmutableList.of(user), VerifierType.InMemory, emptyMap()).get();
```
Comment (rnicoll) [inline java-source/src/main/java/com/template/Main.java]: Sorry, should have said these will need the ".example" TLD added as well

Comment (exFalso) [inline java-source/src/main/java/com/template/Main.java]: Sorry i didn't read this code before submitting the PR, just applied a diff Chris sent me:D

Comment (chrisr3) [inline java-source/src/main/java/com/template/Main.java]: You say that like it's an excuse ... ;-)

Comment (exFalso) [inline java-source/src/main/java/com/template/Main.java]: It is haha

**Comment by exFalso** [comment]
@rnicoll is this good to go?

**Comment by rnicoll** [comment]
Yup, go for it!

**Pull request merged**